### PR TITLE
Add support for arm64 machine type

### PR DESCRIPTION
**Pull request by peterkraume**
This patch adds support for Apple M2 Pro

Fixes: #270

### CODE REVIEW

**Comment by janedbal** [comment]
Thanks, are you able to fix CI?

**Comment by peterkraume** [comment]
CI is fixed now

File: src/EditorconfigChecker/Utilities.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ public static function downloadReleaseArchive(string $releaseName, string $versi
 
         $releaseSuffix = '.tar.gz';
         // Windows release archive suffix changed from 2.5 to 2.6
-        if (self::getCurrentOs() === 'windows' && version_compare(CORE_VERSION, '2.6.0', '<')) {
+        if (self::getCurrentOs() === 'windows') {
```
Comment (janedbal) [inline src/EditorconfigChecker/Utilities.php:83]: I believe this is incorrect and should stay as it was. Now you change the suffix to `.exe.tar.gz` for all windows releases which is wrong.

See 2.5 and 2.6 releases https://github.com/editorconfig-checker/editorconfig-checker/releases

**Review by janedbal — CHANGES_REQUESTED** [review]

File: src/EditorconfigChecker/Utilities.php (hunk 1 above)
Comment (peterkraume) [reply src/EditorconfigChecker/Utilities.php:83]: PHPStan complained that right of the && is always false. That's why  I removed it. CORE_VERSION is now set to 2.7.0.

Comment (janedbal) [reply src/EditorconfigChecker/Utilities.php:83]: I see it now, they change the suffix in main package to and fro :facepalm: 

- 2.7.0: ec-windows-386.exe.tar.gz
- 2.6.0: ec-windows-386.tar.gz
- 2.5.0: ec-windows-386.exe.tar.gz

So this change would broke it for 2.6 (as stated in the comment), but actually works for 2.7.0. 

---------

So please just remove that misleading comment `// Windows release archive suffix changed from 2.5 to 2.6` and I think we can merge it.

Comment (peterkraume) [reply src/EditorconfigChecker/Utilities.php:82]: ✅

**Comment by peterkraume** [comment]
Is there anything left that blocks the merge of this pull request? I'd love to see a new version tagged 👍

**Review by janedbal — APPROVED** [review]

**Pull request merged**